utils/spiel/m0spiel

```diff
@@ -581,7 +581,7 @@ class SpielWrapper:
 
     def __net_domain_init(self):
         domain = self.__malloc(m0_net_domain__size())
-        xprt = NetXprt.in_dll(self.motr, 'm0_net_lnet_xprt')
+        xprt = NetXprt.in_dll(self.motr, 'm0_net_xprt_obj')
```

Comment:
I am sorry, but I didn't see the "m0_net_xprt_obj" is defined or used in anywhere else.
What is it?

Reply:
please run this `./spiel/st/m0t1fs_spiel_sns_repair.sh `

This will give error:
```
lt-m0ham: Connecting to 172.16.154.179@tcp:12345:34:1.
lt-m0ham: Got entrypoint reply.
lt-m0ham: Sent message #2.
lt-m0ham: Finishing.
lt-m0ham: Message #2 is delivered.
echo "!ha_msg
fid: !fid [0,0]
process: ^r|0.0
service: ^s|0.0
data:  !ha_nvec_set
  ignore_same_state: false
  notes:
    - {fid: ^k|1.1, state: repair}
    - {fid: ^k|1.9, state: repair}" | /work/cortx-motr/utils/m0hagen | /work/cortx-motr/utils/m0ham -v -s 172.16.154.179@tcp:12345:33:1003 172.16.154.179@tcp:12345:33:1
lt-m0ham: Connecting to 172.16.154.179@tcp:12345:33:1.
lt-m0ham: Got entrypoint reply.
lt-m0ham: Sent message #2.
lt-m0ham: Finishing.
lt-m0ham: Message #2 is delivered.
/work/cortx-motr/utils/spiel/m0spiel -l /work/cortx-motr/motr/.libs/libmotr.so --client 172.16.154.179@tcp:12345:34:1001 --ha 172.16.154.179@tcp:12345:34:1
Traceback (most recent call last):
  File "/work/cortx-motr/utils/spiel/m0spiel", line 720, in <module>
    tool.spiel.spiel_init(args.client, args.ha)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 199, in spiel_init
    self.__net_domain_init()
  File "/work/cortx-motr/utils/spiel/m0spiel", line 584, in __net_domain_init
    xprt = NetXprt.in_dll(self.motr, 'm0_net_xprt_obj')
ValueError: /work/cortx-motr/motr/.libs/libmotr.so: undefined symbol: m0_net_xprt_obj
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
Error in sys.exitfunc:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
Abort SNS repair (1).
/work/cortx-motr/utils/spiel/m0spiel -l /work/cortx-motr/motr/.libs/libmotr.so --client 172.16.154.179@tcp:12345:34:1001 --ha 172.16.154.179@tcp:12345:34:1
Traceback (most recent call last):
  File "/work/cortx-motr/utils/spiel/m0spiel", line 720, in <module>
    tool.spiel.spiel_init(args.client, args.ha)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 199, in spiel_init
    self.__net_domain_init()
  File "/work/cortx-motr/utils/spiel/m0spiel", line 584, in __net_domain_init
    xprt = NetXprt.in_dll(self.motr, 'm0_net_xprt_obj')
ValueError: /work/cortx-motr/motr/.libs/libmotr.so: undefined symbol: m0_net_xprt_obj
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
Error in sys.exitfunc:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
wait for sns repair (1).
/work/cortx-motr/utils/spiel/m0spiel -l /work/cortx-motr/motr/.libs/libmotr.so --client 172.16.154.179@tcp:12345:34:1001 --ha 172.16.154.179@tcp:12345:34:1
Traceback (most recent call last):
  File "/work/cortx-motr/utils/spiel/m0spiel", line 720, in <module>
    tool.spiel.spiel_init(args.client, args.ha)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 199, in spiel_init
    self.__net_domain_init()
  File "/work/cortx-motr/utils/spiel/m0spiel", line 584, in __net_domain_init
    xprt = NetXprt.in_dll(self.motr, 'm0_net_xprt_obj')
ValueError: /work/cortx-motr/motr/.libs/libmotr.so: undefined symbol: m0_net_xprt_obj
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
Error in sys.exitfunc:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/work/cortx-motr/utils/spiel/m0spiel", line 207, in spiel_fini
    self.motr.m0_spiel_fini(self.spiel)
AttributeError: SpielWrapper instance has no attribute 'spiel'
Failed: SNS repair failed..
/var/motr/systest-2506
unmounting and cleaning..
```